java-ast/addon/src/main/java/org/jboss/windup/ast/java/JavaASTProcessor.java

```diff
@@ -199,43 +201,55 @@ private JavaASTProcessor(WildcardImportResolver importResolver, CompilationUnit
 
         cu.accept(this);
     }
+    
+    private String extractDefinitionLine(String typeDeclaration) {
+        String typeLine = "";
+        String[] lines = typeDeclaration.split(System.getProperty("line.separator"));
```

Comment:
as jess/lincoln said, this is system specific and will not work in case the scanned library comes from different system